sdk/push_client.go

```diff
@@ -85,6 +85,9 @@ func (c *PushClient) PublishMultiple(messages []PushMessage) ([]PushResponse, er
 }
 
 func (c *PushClient) publishInternal(messages []PushMessage) ([]PushResponse, error) {
+	// Used for sanity check
+	var exptectedReceipts int = 0
```

Comment:
```suggestion
	var expectedReceipts int = 0
```
There's a small typo in this variable's name.